app/views/articles/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<% @articles.each do |article| %>
+  <div id=header>
+    <p><%= article.header %></p>
+  </div>
+
+  <div id=subheader>
+    <p><%= article.subheader %></p>
+  </div>
+
+  <div id=subheader>
+    <p><%= article.body %></p>
+  </div>
+
+<% end %>
+
+<%=image_tag("trump.jpg")%>
```

Comment:
Should we perhaps have this inside the div above? that way each article gets a picture placholder.